basic_integral/basic_integral.cpp

```diff
@@ -0,0 +1,24 @@
+#include <iostream>
+#include <math.h>
+
+using namespace std;
+
+int main(){
+    cout << "Iterations ?";
+    int n;
+    cin >> n;
+    int N=n;
```

Comment:
Why would you duplicate variable n?

Reply:
its not duplicate they are different . 
n is Iteration times but N is upper limit of sigma